securedrop_client/gui/widgets.py

```diff
@@ -161,6 +161,20 @@ def update_view(self, widget):
         widget.setVisible(True)
         self.view_layout.addWidget(widget)
 
+    def re_create_conversation_view(self):
+        """Take old conversation view off, and re-create it
+
+        The re created conversation view is blank(without any conversation).
+        """
+        deprecated_widget = self.layout.takeAt(1)
+        if deprecated_widget:
+            deprecated_widget.widget().setVisible(False)
+            self.view_holder = QWidget()
+            self.view_layout = QVBoxLayout()
+            self.view_holder.setLayout(self.view_layout)
+            self.view_holder.setVisible(True)
+            self.layout.addWidget(self.view_holder, 6)
```

Comment:
What is the 6? 

For constants (provided they are indeed constant and do not vary depending on the application's state), let's define them as such so that it is clear what they are. This is in the spirit of avoiding [magic numbers](https://facweb.northseattle.edu/voffenba/class/csc142/NoMagicNumbers.html) in the codebase. We should define a constant for the 6 (and the 1 above in the `deprecated_widget` variable definition).